lib/services/profiles-api-service.js

```diff
@@ -76,16 +76,14 @@ function profileApiServiceFactory(
     };
 
     ProfileApiService.prototype.setLookup = function(req) {
-        var query = req.query;
+        var query = req.swagger.query;
```

Comment:
@heckj I think we want to check whether the swagger object is defined in the request so that the profiles service will work with both the 1.1 and 2.0 APIs.  I think this will fix your test failure as well.
```
var query = req.swagger ? req.swagger.query : req.query;
```